components/GitHubAccelerator.js

```diff
@@ -8,11 +8,12 @@ export default function GitHubAccelerator() {
     <div className="mb-12">
       <div>
         <Image
-          className="h-32 w-full object-cover lg:h-48"
+          className="h-32 w-full lg:h-48"
```

Comment:
Just add the contain property here.

```suggestion
          className="h-32 w-full lg:h-48 object-contain"
```